main.js

```diff
@@ -68,6 +70,26 @@ module.exports = function(options) {
 	var sensuChecks = sensu(name, options.sensuChecks);
 	var healthChecks = options.healthChecks;
 
+	// Note: Only *report* CSP violations.
+	// The aim is to know which scripts are being executed via third-parties.
+	if (options.withContentSecurityPolicy) {
+		csp.extend(app, {
+			reportPolicy: {
+				directives: {
+					'report-uri': ['https://ft-next-csp-ingest.herokuapp.com/__csp-ingest'],
```

Comment:
ahh please use .ft.com URLs (either this should be sent through the router on `next.ft.com` or should have its own domain (see konstructor.svc.ft.com to set up DNS and Fastly.com to do the SSL termination) …… sorry :see_no_evil: